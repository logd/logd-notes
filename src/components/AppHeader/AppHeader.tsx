import React from "react";
import styled from "styled-components";
import { Link, useLocation } from "react-router-dom";
import { UserNav } from "../UserNav/UserNav";

const theme = `
    font-size: .9em;
    font-weight: 100;
    font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
`;

const Wrapper = styled.div`
  display: flex;
  padding: 1em;
  ${theme};
`;

// const Smaller = styled.div`
//   font-size: smaller;
// `;

const Main = styled.div`
  flex: 1;
  color: #c9d6df;
`;
export const AppHeader = () => {
  const location = useLocation();
  const isHome = location.pathname === "/";

  return (
    <Wrapper>
      <Main>{isHome ? <>logd</> : <Link to="/">logd</Link>}</Main>
      {/* <Smaller>Quick personal notes, saved locally only.</Smaller> */}
      <UserNav />
    </Wrapper>
  );
};
