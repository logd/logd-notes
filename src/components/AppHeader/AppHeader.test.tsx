import React from "react";
import renderer from 'react-test-renderer';
import { AppHeader } from "./AppHeader";

it("renders correctly", () => {
  const tree = renderer
    .create(<AppHeader />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
