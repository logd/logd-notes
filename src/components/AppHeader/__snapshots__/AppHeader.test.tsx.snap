// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-AxjAm gBVPNU"
>
  <div
    className="sc-AxiKw cjyCOd"
  >
    logd
  </div>
  <div
    className="sc-AxirZ gnYjwA"
  >
    Quick personal notes, saved locally only.
  </div>
  <div>
    <button
      onClick={[Function]}
    >
      Login
    </button>
  </div>
</div>
`;
